docs/source/api/artifacts.rst

```diff
@@ -65,3 +65,7 @@ TextFileArtifact
 ++++++++++++++++
 .. autoclass:: bentoml.service.artifacts.common.TextFileArtifact
 
+TransformersModelArtifact
```

Comment:
👍 

could you also add Transformers in the main `README.md`'s frameworks section?